renv.lock

```diff
@@ -940,7 +946,7 @@
       "Package": "lubridate",
       "Version": "1.9.3",
```

Comment:
1.9.4 [appears to be available on CRAN](https://cran.r-project.org/web/packages/lubridate/index.html), but [its URL cannot be resolved](https://github.com/ccao-data/model-condo-avm/actions/runs/12240335482/job/34143092408#step:7:2329). Rather than spend more time investigating this discrepancy, we pin to 1.9.3.